indico/modules/events/editing/client/js/models.js

```diff
@@ -25,3 +27,11 @@ export const EditingReviewAction = {
   update: 'update',
   requestUpdate: 'request_update',
 };
+
+export const EditableTypeTitles = {
+  paper: Translate.string('Paper'),
+  slides: Translate.string('Slides'),
+  poster: Translate.string('Poster'),
+};
+
+export const EditableTypeOrder = ['paper', 'slides', 'poster'];
```

Comment:
`editableTypeOrder`? (the other names are using UpperCase to match the enums, but this one doesn't match an enum)